content/en/docs/concepts/security/index.md

```diff
@@ -487,7 +487,7 @@ policies:
   workloads  switch successfully to the desired mode, you can apply the policy
   with the final mode. You can use Istio telemetry to verify that workloads
   have switched successfully.
-- When migrating peer authentication policies from one JWT to another, add
+- When migrating request authentication policies from one JWT to another, add
```

Comment:
This line is not zh translation, but instead an en doc change, so it should be made in a separate en PR.

Reply:
pr https://github.com/istio/istio.io/pull/7009